NETPROGRAM/ArrayExercise/Exercise07.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NETPROGRAM.ArrayExercise
+{
+    public class Exercise07
+    {
+
+        public List<IGrouping<int,int>> SplitArray(int[] array, int n)
```

Comment:
Phải trả về như thế này

```suggestion
        public List<List<int>> SplitArray(int[] array, int n)
```

Reply:
done